src/Microsoft.AspNetCore.Razor.Language/Syntax/NodeFlags.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Razor.Language
```

Comment:
Should these things be in a `.Syntax` namespace?

Reply:
They could. I don't have a specific reason for not doing that.